test/swim_test.js

```diff
@@ -181,6 +187,8 @@ test('marks member faulty after suspect period', function t(assert) {
 
     suspicion.start(member);
 
-    suspicion.stopAll();
-    assert.end();
+    setTimeout(function verify() {
```

Comment:
mocking out setTimeout used by suspicion not only does not return a timeout object, also behaves differently from code running in prod

by passing suspicionTimeout in test, suspicion would behave close to prod and this test still runs deterministically